rosbag2_py/src/rosbag2_py/format_bag_metadata.cpp

```diff
@@ -118,10 +119,14 @@ void format_topics_with_type(
   }
 
   auto print_topic_info =
-    [&info_stream](const rosbag2_storage::TopicInformation & ti) -> void {
+    [&info_stream, &topics_size](const rosbag2_storage::TopicInformation & ti) -> void {
       info_stream << "Topic: " << ti.topic_metadata.name << " | ";
       info_stream << "Type: " << ti.topic_metadata.type << " | ";
       info_stream << "Count: " << ti.message_count << " | ";
+      auto topic_size_iter = topics_size.find(ti.topic_metadata.name);
+      if (topic_size_iter != topics_size.end()) {
+        info_stream << "Size Contribution: " << format_file_size(topic_size_iter->second) << " | ";
+      }
       info_stream << "Serialization Format: " << ti.topic_metadata.serialization_format;
       info_stream << std::endl;
     };
```

Comment:
It would be great to have consistent output about `Size contribution: ` and print it out even for topics with zero messages and zero size contribution. 
We can distinguish verbose/no-verbose cases by checking `topics_size.size()` or, even better, by adding an extra `bool verbose` parameter, to the function for future extensions. Note: we shall not print `Size contribution: ` in non-verbose mode.

The rationale for the consistent output is that some external script that could parse output from the `ros2 bag info --verbose` can rely on the `Size contribution: ` field to be present for each topic.

Reply:
Done, and also added size contribution for services.